package.json

```diff
@@ -68,7 +69,8 @@
     "tsd": "^0.24.1",
     "typescript": "^4.8.4",
     "typescript-cached-transpile": "0.0.6",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "v8-profiler-next": "^1.9.0"
```

Comment:
This seems to have added a lot to the lock file I'm happy to make this a "must install before launching benchmaks" style dep, much like the other versions of the BSON lib are installed with --no-save

Reply:
no strong preferences here since it's a dev dependency; though if we're intending to run perf tests in the pipeline, may be worth keeping just to avoid surprise failures